src/style-spec/reference/v8.json

```diff
@@ -811,7 +811,8 @@
       "function": "interpolated",
       "zoom-function": true,
       "property-function": true,
-      "doc": "Scale factor for icon. 1 is original size, 3 triples the size.",
+      "units": "factor of the original icon size",
+      "doc": "Scales the original size of the icon by the provided factor. The new pixel size of the image will be `original pixel size * 'image-size'`. 1 is the original size, 3 triples the size of the image.",
```

Comment:
Tweak the second sentence so that only `image-size` is in backticks. This ensures that the macOS/iOS API documentation generator will replace "image-size" with the SDK-appropriate property. cc @1ec5